LocalSettings.php

```diff
@@ -1794,8 +1794,8 @@
 		],
 		'isvwiki' => [
 			'status' => [
+				'quality' => 1,
```

Comment:
Quality will be removed soon once MediaWiki 1.36 is deployed, FYI. See https://phabricator.wikimedia.org/T277883.

Reply:
https://noc.wikimedia.org/conf/highlight.php?file=flaggedrevs.php still has them while they are at MediaWiki 1.37, so I am not sure about that. Either way, the whole config would need fixing then, I'm just trying to learn if this causes the issue of no autoreview.